src/component/list.js

```diff
@@ -61,6 +62,7 @@ List.propTypes = {
   data: PropTypes.array.isRequired,
   renderHeader: PropTypes.func,
   renderItem: PropTypes.func.isRequired,
+  stickyHeaderIndices: PropTypes.array,
```

Comment:
`stickyHeaderIndices` was added as a prop because I assumed we might not want to have sticky headers on all lists. If that's not the case we can hardcode `stickyHeaderIndices={[0]}` instead of line 53.

Reply:
IMO we do always want sticky headers :)

Reply:
Cool, I'll make the appropriate changes.

Reply:
@valentinewallace updated - all lists now have sticky headers.